reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -316,166 +352,127 @@ def system_equilibrated(job):
                 print(
-                    "System {} is equilibrated at cycle {}. Completed {} equil loops".format(
-                        job,
-                        max(
-                            equil_status_length1[1],
-                            equil_status_energy1[1],
-                            equil_status_length2[1],
-                            equil_status_energy2[1],
-                            equil_status_toal_energy[1],
-                        ),
-                        job.doc.get("equil_replicates_done"),
+                    "Equil status total energy={}".format(
+                        equil_status_total_energy
                     )
                 )
-                text_file = open("equil_information.txt", "w")
-                n = text_file.write(
-                    "System {} is equilibrated at cycle {}. Completed {} equil loops".format(
-                        job,
-                        equil_status[1],
-                        job.doc.get("equil_replicates_done"),
+                if len(files) >= 3:
+                    print(
+                        "Even though the system is not equilibrated according to pymbar, we are considering this system equilibrated as 3 equil loops are completed"
                     )
-                )
-                text_file.close()
-                return True
-
-        if job.sp.ensemble == "GEMC-NVT":
-            equil_log_box1 = sanitize_gemc_log("equil")[0]
-            equil_log_box2 = sanitize_gemc_log("equil")[1]
-            equil_status_length1 = is_equilibrated(
-                equil_log_box1[:, 0], threshold=0.2, nskip=100
-            )
-            equil_status_energy1 = is_equilibrated(
-                equil_log_box1[:, 3], threshold=0.2, nskip=100
-            )
-            equil_status_length2 = is_equilibrated(
-                equil_log_box2[:, 0], threshold=0.2, nskip=100
-            )
-            equil_status_energy2 = is_equilibrated(
-                equil_log_box2[:, 3], threshold=0.2, nskip=100
-            )
-            equil_status_total_energy = is_equilibrated(
-                equil_log_box1[:, 3] + equil_log_box2[:, 3],
-                threshold=0.2,
-                nskip=100,
-            )
-            if (
-                equil_status_length1[0]
-                and equil_status_energy1[0]
-                and equil_status_length2[0]
-                and equil_status_energy2[0]
-                and equil_status_total_energy
-            ) == False:
-                print(
-                    "System {} is not equilibrated. Completed {} equil loops".format(
-                        job, job.doc.get("equil_replicates_done")
+                    text_file = open("equil_information.txt", "w")
+                    n = text_file.write(
+                        "Even though the system is not equilibrated according to pymbar, we are considering this system equilibrated as 3 equil loops are completed"
                     )
```

Comment:
I would convert this to a `with` statement too
```suggestion
                    with open("equil_information.txt", "w") as text_file:
                        n = text_file.write(
                            "Even though the system is not equilibrated according to pymbar, we are considering this system equilibrated as 3 equil loops are completed"
                    )
```